src/Header/RequestHandler.h

```diff
@@ -19,12 +19,19 @@
 class RequestHandler
 {
 public:
-    //instantiate instance with request, will parse to OperateData and check with Authenticator and call correspond api.
-    RequestHandler(const std::string &requestData);
+    //instantiate instance with accepted file descriptor, will receive request string and parse to OperateData, then check with Authenticator and call correspond api.
+    RequestHandler(const int &file_descriptor);
+    //receive request string and send to api
+    void RequestHandler::WaitForMessage();
```

Comment:
You don't need to add `RequestHandler::` in the definition.

Reply:
Done